src/Synapse/Synapse/help/Get-AzSynapseSqlPoolGeoBackup.md

```diff
@@ -0,0 +1,122 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Synapse.dll-Help.xml
+Module Name: Az.Synapse
+online version: https://docs.microsoft.com/en-us/powershell/module/az.synapse/get-azsynapsesqlpoolgeobackup
+schema: 2.0.0
+---
+
+# Get-AzSynapseSqlPoolGeoBackup
+
+## SYNOPSIS
+Gets a geo-redundant backup of a Sql Pool.
+
+## SYNTAX
+
+### GetByNameParameterSet (Default)
+```
+Get-AzSynapseSqlPoolGeoBackup [-ResourceGroupName <String>] -WorkspaceName <String> -Name <String>
```

Comment:
Please also provide getbyresourceId parameter set for Get-AzSynapseSqlPoolGeoBackup

Reply:
I think we dont have it when I refer to sql commands. Please feel free to let me know if I'm wrong